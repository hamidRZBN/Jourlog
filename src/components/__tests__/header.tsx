import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Header } from '../';

afterEach(cleanup);

it('renders', () => {
  const { asFragment } = render(<Header>Header</Header>);
  expect(asFragment()).toMatchSnapshot();
});
