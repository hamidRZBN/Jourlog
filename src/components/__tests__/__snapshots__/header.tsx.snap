// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <header
    class="sc-AykKI dgloPn"
  >
    <div
      class="sc-AykKJ bWHXCe"
    >
      Header
    </div>
  </header>
</DocumentFragment>
`;
